scripts/build_nginx

```diff
@@ -36,6 +36,26 @@ echo "Downloading $zlib_url"
 echo "Downloading $uuid4_url"
 (cd nginx-${NGINX_VERSION} && curl -L $uuid4_url | tar xvz )
 
+if [ -d "/buildpack/support/patchfiles/${NGINX_VERSION}" ]
+then
+    PATCHFILES=$(find /buildpack/support/patchfiles/${NGINX_VERSION} -name '*.patch')
+else
+    exit 1
+fi
+
+
+NGX_SHUTDOWN_SIGNAL=TERM
+NGX_TERMINATE_SIGNAL=QUIT
```

Comment:
I'm pretty sure these two lines aren't doing anything, since they're just setting shell variables that are never used.  I suspect a previous version of this script used them, but they're no longer necessary.